src/components/Backdrop/styles.ts

```diff
@@ -1,15 +1,17 @@
 import { Box } from "@chakra-ui/layout";
 import styled from "styled-components";
 
-export const StyledBackdrop = styled(Box)<{isWhite: boolean}>`
+export const StyledBackdrop = styled(Box)<{iswhite: boolean}>`
   position: absolute;
   top: 0;
   left: 0;
   height: 100vh;
   width: 100%;
-  background-color: ${props => props.isWhite ? "var(--color-white)}" : "var(--color-black)"};
-  opacity: ${props => !props.isWhite && 0};
-  animation: ${props => !props.isWhite && "appear .5s ease"};
+  background-color: ${props => props.iswhite ? "var(--color-white)}" : "var(--color-black)"};
+  opacity: ${props => !props.iswhite && 0};
+  animation: ${props => !props.iswhite && "appear .5s ease"};
```

Comment:
it's ugly, but if you don't want to have the animation attribute when "isWhite", you can do something like
https://styled-components.com/docs/basics#pseudoelements-pseudoselectors-and-nesting